source/documentation/infrastructure/getting-access.md

```diff
@@ -0,0 +1,113 @@
+# Accessing the Infrastructure
+
+## VPN
+
+All connections must be made via the GDS VPN. Please contact your local service desk for access.
+
+## Bastions
+
+The bastion servers act as a gateway to their respective clusters + databases.
+That is to say:
+
+- To access any Staging database or server, you must access via the staging bastion.
+- To access any Production database or server, you must access via the production bastion.
+
+### Accessing the Bastion
+
+#### Secrets
+
+[Extract the SSH Key secrets][getting-a-secret] for the bastion server:
+
+- staging secret name: `keys/govwifi-staging-bastion-key`
+- production secret name: `keys/govwifi-bastion-key`
+
+Find out the Elastic IPs for the bastion servers. You can do this by going into the AWS console,
+and find the instances with the Bastion name in.
+
+Remember that there are 2 regions, so there may be more than 2 bastions.
+
+#### SSH Config
+
+It is recommended to set up an ssh config for ease of use. All further instructions will
+assume you use similar naming.
+
+**Note**: The IP addresses have been redacted. Please substitute in the correct IP addresses.
```

Comment:
Do you already have an example of an ssh-config in the repo somewhere? If so, you can link to it instead

Reply:
nope, this is the only example